### ISSUE TITLE
Nipype Interfaces

### ISSUE THREAD

**Issue by chasejohnson3** [nipype/pydra#407]
I am trying to run antsRegistration as part of a pipeline using pydra. Is there support and/or examples of using the interfaces from nipype.interfaces.ants in pydra?

**Comment by effigies** [comment]
The [pydra-nipype1](https://github.com/nipype/pydra-nipype1) package is intended to support use cases like this. There's an example in the docstring:

https://github.com/nipype/pydra-nipype1/blob/4c0bea186a11ede1beb670cab6c2dcf352acc3de/pydra/tasks/nipype1/utils.py#L20-L41

Note that workflow build-time type checking does not work; types can only be checked at runtime.

**Comment by chasejohnson3** [comment]
Perfect, that is what I was looking for

**Comment by chasejohnson3** [comment]
@effigies I was able to use the pydra-nipype1 package successfully for AntsRegistration and AntsApplyTransforms. However, I am now trying to apply pydra-nipype1 to AntsJointFusion, and while the cmdline used by the nipype interface runs without issue by itself, the nipype interface gets hung up. 

This is probably a nipype interface issue rather than an issue with pydra-nipype1 (I posted it in [nipype issues #3313](https://github.com/nipy/nipype/issues/3313)), but I am curious if this issue of the cmdline working without the nipype interface working has been observed before with AntsJointFusion or other nipype interfaces.
